packages/pass-style/src/deeplyFulfilled.js

```diff
@@ -73,7 +58,7 @@ const { fromEntries } = Object;
  * is for the higher "@endo/patterns" level of abstraction to determine,
  * because it defines the `Key` notion in question.
  *
- * @template {Passable} [T=Passable]
+ * @template {Passable} T
```

Comment:
Is there a reason to remove the default?

In general when I write a type-constrained type parameter, I also provide that type constraint as a default binding of the type parameter. Should I stop doing so?

Reply:
For generic functions we want to rely on inference. I don't remember exactly why I dropped it, but I think a default doesn't have any advantages here, especially one where the default value is the constraint itself.

For type definitions it's a different story as it allows you to reference the bare type without type parameters.